### ISSUE TITLE
Allow users to change name(s)

### ISSUE THREAD

**Issue by mikekasprzak** [ludumdare/ludumdare#1621]
Allow users to change their.

* Name
* Slug
* "Real Name"

A significant amount of support requests is this.

**Comment by elitegoliath** [comment]
Is this still a priority?

**Comment by mikekasprzak** [comment]
Yes, once I have time to work on features. Unfortunately I've been swamped
with work (ugh, launching a game), and I need to make time fix the server
uptime before the next event.

**Comment by elitegoliath** [comment]
If I can get squared away with some of my other projects, I may be able to lend a hand. Thank you for the amazing work you do btw.

**Comment by LoneWolfHT** [comment]
I shall look forward to testing this out when you implement it :P
